flashmodels/patch/llama_model.py

```diff
@@ -394,9 +395,80 @@ def forward(
         return outputs
 
 
+def spmd_flash_attn_fwd(
```

Comment:
Can we use only one flash_attn_fwd patch function by only adding two arguments like `use_spmd`, `fsdp_num`?